ansible/files/prosody.cfg.lua

```diff
@@ -75,6 +75,7 @@ modules_enabled = {
 		"update_notify";
 		"turncredentials";
 		"admin_shell";
+		"isolate_host";
```

Comment:
How does this only take effect for restricted users? I can't see that yet from the configuration. Is this a thing in mod_isolate_host now?

Reply:
mod_isolate_host sets a per-session flag to indicate the isolation status. I created a custom mod_snikket_restricted_users which sets this flag appropriately based on the user's role.

Reply:
I just committed some explanatory comments.

Reply:
:+1: Adding explanations as comments is my favourite reaction to review remarks! :)